src/options.go

```diff
@@ -1333,6 +1355,32 @@ func parseOptions(opts *Options, allArgs []string) {
 			}
 		}
 	}
+
+	if validatePointer {
+		if err := validateSign(opts.Pointer, "pointer"); err != nil {
+			errorExit(err.Error())
+		}
+	}
+
+	if validateMarker {
+		if err := validateSign(opts.Marker, "marker"); err != nil {
+			errorExit(err.Error())
+		}
+	}
+}
+
+func validateSign(sign string, signOptName string) error {
+	widthSum := 0
+	for _, r := range sign {
+		if !unicode.IsGraphic(r) {
+			return fmt.Errorf("invalid character in %v", signOptName)
+		}
+		widthSum += runewidth.RuneWidth(r)
+	}
+	if widthSum > 2 {
+		return fmt.Errorf("%v display width should be up to 2", signOptName)
+	}
```

Comment:
We can put `if` statement inside `for` loop for early termination.

Reply:
Thank you.

fixed in (ddc7435)